### PR TITLE
Create Dockerfile-No-Jdk

### PR DESCRIPTION
**Pull request by anjia0532**
ref doc https://alibaba.github.io/arthas/en/docker.html#install-arthas-into-the-base-docker-image

In this scenario
base on openjdk:8-alpine, the image size > 110MB
```
docker build --build-arg MIRROR=true . -t arthas:jdk
```
base on alpine only 15MB
```
docker build -f Dockerfile-No-Jdk --build-arg MIRROR=true . -t arthas:no-jdk
```
image

### CODE REVIEW

File: Dockerfile-No-Jdk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+ENV MAVEN_HOST=https://repo1.maven.org/maven2 \
+    ALPINE_HOST=dl-cdn.alpinelinux.org \
+    MIRROR_MAVEN_HOST=http://maven.aliyun.com/repository/public \
+    MIRROR_ALPINE_HOST=mirrors.aliyun.com 
+
+# if use mirror change to aliyun mirror site
+RUN if $MIRROR; then MAVEN_HOST=${MIRROR_MAVEN_HOST} ;ALPINE_HOST=${MIRROR_ALPINE_HOST} ; sed -i "s/dl-cdn.alpinelinux.org/${ALPINE_HOST}/g" /etc/apk/repositories ; fi && \
+    # https://github.com/docker-library/openjdk/issues/76
+    apk add --no-cache tini && \
```
[the remark this answers is not in the document]
Comment (anjia0532) [reply Dockerfile-No-Jdk]: yeap, wait a moment,delete it.

[the remark this answers is not in the document]
Comment (anjia0532) [reply Dockerfile-No-Jdk]: done

File: Dockerfile-No-Jdk [hunk 2]
```diff
@@ -4,19 +4,14 @@ ARG ARTHAS_VERSION="3.0.5"
 ARG MIRROR=false
 
 ENV MAVEN_HOST=https://repo1.maven.org/maven2 \
-    ALPINE_HOST=dl-cdn.alpinelinux.org \
-    MIRROR_MAVEN_HOST=http://maven.aliyun.com/repository/public \
-    MIRROR_ALPINE_HOST=mirrors.aliyun.com 
+  ALPINE_HOST=dl-cdn.alpinelinux.org \
```
Comment (hengyunabc) [inline Dockerfile-No-Jdk]: `ALPINE_HOST` can be deleted?

**Comment by hengyunabc** [comment]
Thank you very much.

**Pull request merged**